exampleCourse/questions/demo/addHTMLtable/server.py

```diff
@@ -0,0 +1,29 @@
+import random
+import io
+
+def generate(data):
+
+    name_list = ["Carla", "Manoel", "Sam", "Laura", "Amanda"]
+    age_list =  [24, 18, 30, 45, 32]
+    name_greek = ["$\\gamma$", "$\\mu$", "$\\sigma$", "$\\lambda$", "$\\alpha$" ]
+
+    # Generate complete html string
+    mytable = '<table style="width:30%"><tr><th> Name </th> <th> Age </th> <th> Id </th></tr>'
+    for i in range(len(age_list)):
+        mytable += '<tr><td>'+name_list[i]+'</td><td>'+str(age_list[i])+'</td><td>'+name_greek[i]+'</td><tr>'
```

Comment:
What do you think of using `zip()` here? i.e.
```python
for name, age, name_greek in zip(name_list, age_list, name_greek):
```
Totally minor nit, but I think it looks a little cleaner

Reply:
haha. Sure. Looks a lot better. Thank you :-)

Would you be able to make these changes and merge? It would help me a lot (getting behind with all the workshop stuff :-( )